guide/admin/homeview.md

```diff
@@ -0,0 +1,33 @@
+## User Handbook
+### Changing the Home Page View
```

Comment:
I'm trying to figure out how to keep both the consistent terminology and try to pick up on key words that are regularly used in support requests. Many are looking for a category view or tags only, but I'm not sure how to word that without overdoing it. 

Maybe we just use what you suggested and see what happens.